### ISSUE TITLE
Slow color change on status change when using powerline themes

### ISSUE THREAD

**Issue by rad4day** [tobi-wan-kenobi/bumblebee-status#355]
#### Summary
Affected module: Core (I guess), Commit: a2cc828 and before probably.

#### Description
When a module changes state, say from warning to critical the powerline characters take around 1 second to switch their color.

#### What is the current behaviour and what is the expected behaviour?
[Screenshot](https://i.imgur.com/7luQNNI.png)

#### How to reproduce
Select a module which can trigger a status change by a mouse click. For example <spaceapi> which changes state to reloading on click, and a powerline theme like gruvbox-powerline. Trigger a status change and see the screenshot abov.e.

I guess this was introduced when you changed the updater code in response to #353

**Comment by tobi-wan-kenobi** [comment]
Yeah, I realized that today, too. Updating the prefix/postfix separators is non-trivial with the current code.

Right now, I'm leaning towards reverting the changes made for #353. Having selective updates is really nice, but I guess status changes are typically more frequent than scrollovers that really hurt in #353. What's your take, as contributor?
